lib/chef/knife/windows_cert_generate.rb

```diff
@@ -75,7 +76,7 @@ def prompt_for_passphrase
       end
 
       def generate_certificate rsa_key
-        @hostname = "*"
+        @hostname = Socket.gethostname
```

Comment:
Ergh. I think I still have an AD controller here I could test this with, but hopefully you can answer this off the top of your head:

Say I have a server. web01.atlanta.eng.contoso.com. As a Windows administrator who might not know where I am in the forest, am I going to run ```knife windows bootstrap winrm web01.atlanta``` because I usually type web01.atlanta into remote desktop? Is that going to work if my CN is web01.atlanta.eng.contoso.com? Or would I need a bunch of SANs for that?